Examples/timberman.md

```diff
@@ -0,0 +1,72 @@
+# Timberman
+
+**Playstore Link:**  [Timberman](https://play.google.com/store/apps/details?id=com.dm.timber&hl=en)
+
+##### About the game:
+Timberman is an oldschool arcade style casual game. You have to put on your checkered shirt, pull up your sleeves, pick up the sharpened ax and ... grow a beard. In a moment there will be splinters flying and knocks off ax will bear echoed through the forest. Show how quick and how efficiently you can cut down a hundred-meter tall pine tree. But be careful, work of a lumberjack is tough and it's not hard for the accident at work. In addition, time is chasing you in every step not giving even a moment to rest. 
+
+##### Difficulty Level : 
+Easy
+
+##### Overview
+As soon as the timber branch appears above the timberman, the voltage output from the pin where the LDR is connected decreases. Whenever a drop in voltage is detected, the output pin is set to LOW and thus the Relay output is connected to ground which is equivalent to a 'Touch' on the other side of the side bearing lumberjack. In the absence of branch overhead, repeated touch is simulated on the same side of lumberjack.
+
+##### Requirements
+* An Android Device with the 'Timberman’' game installed on it. (Turn on the Developer options for better visualization)
+* Arduino, Breadboard, 22k Resistor X 2, LDR, Relay X 2, Connecting wires, two conducting metal coin
+* Computer to program Arduino
+
+##### Block Diagram
```

Comment:
Learn to add images from here: https://github.com/GameAutomators/eBook-Source/blob/master/Examples/05-ready-steady-bang.md